tests/e2e/utils/set-site-verification.js

```diff
@@ -0,0 +1,14 @@
+
+import { wpApiFetch } from './wp-api-fetch';
+
+/**
+ *
+ * @param {*} config
+ */
+export async function setSiteVerification( verified = true ) {
```

Comment:
Excellent! These new utilities can be used in the new tests added in #393.

Reply:
@lucymtc ☝️

Reply:
Yeah I already mentioned it there :-)

Merged the PR now, so this can be proceeded with.